### PR TITLE
Add support for symfony 7

### LINKED ISSUE #194: Feat: Symfony 7
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the new feature

This package should support symfony 7.

I'm going to work on a PR.

### Is your feature request related to a problem? (optional)

_No response_

### Describe alternatives you've considered (optional)

_No response_

### Additional context (optional)

_No response_

### PR DESCRIPTION
**Pull request by W0rma**
# Description

Please include a summary of the change and which issue is fixed (if any).

Fixes #194 

## Type of change

- [x] Dependency update

# How Has This Been Tested?

- [x] Unit tests

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/tests-8.x.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ['8.0', '8.1']
+        version: ['8.0', '8.1', '8.2', '8.3']
```
Comment (RikudouSage) [inline .github/workflows/tests-8.x.yaml]: The latest version shouldn't be there, only the older ones:

```suggestion
        version: ['8.0', '8.1', '8.2']
```

Comment (W0rma) [reply .github/workflows/tests-8.x.yaml]: Thank you for the hint. Fixed.

File: .github/workflows/tests-8.x.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ['8.0', '8.1']
+        version: ['8.0', '8.1', '8.2']
```
Comment (RikudouSage) [inline .github/workflows/tests-8.x.yaml]: Sorry about that, the 8.2 can't be there either, I thought I've already migrated the codebase to 8.3, in that case it should be there, but as it is, only 8.0 and 8.1 should be there. The version this is built against is tested in `tests.yaml`.

```suggestion
        version: ['8.0', '8.1']
```

Comment (W0rma) [reply .github/workflows/tests-8.x.yaml]: No problem, fixed.

Thank you for clarification.

**Review by RikudouSage — APPROVED** [review]

**Pull request merged**